statusHandler/quotaStatusHandler/printQuotaStatusHandler.go

```diff
@@ -0,0 +1,158 @@
+
+	pqsh.mutStatistics.Lock()
+	defer pqsh.mutStatistics.Unlock()
+
+	numStatistics := len(pqsh.statistics)
+
+	if numStatistics == 0 {
+		log.Trace("empty quota statistics")
+		return
+	}
+
+	for name, q := range pqsh.statistics {
+		avgQuota.numReceivedMessages += q.numReceivedMessages
+		avgQuota.sizeReceivedMessages += q.sizeReceivedMessages
+		avgQuota.numProcessedMessages += q.numProcessedMessages
+		avgQuota.sizeProcessedMessages += q.sizeProcessedMessages
+
+		minQuota.numReceivedMessages = pickMinUint32(minQuota.numReceivedMessages, q.numReceivedMessages)
+		minQuota.sizeReceivedMessages = pickMinUint64(minQuota.sizeReceivedMessages, q.sizeReceivedMessages)
+		minQuota.numProcessedMessages = pickMinUint32(minQuota.numProcessedMessages, q.numProcessedMessages)
+		minQuota.sizeProcessedMessages = pickMinUint64(minQuota.sizeProcessedMessages, q.sizeProcessedMessages)
+
+		maxQuota.numReceivedMessages = pickMaxUint32(maxQuota.numReceivedMessages, q.numReceivedMessages)
+		maxQuota.sizeReceivedMessages = pickMaxUint64(maxQuota.sizeReceivedMessages, q.sizeReceivedMessages)
+		maxQuota.numProcessedMessages = pickMaxUint32(maxQuota.numProcessedMessages, q.numProcessedMessages)
+		maxQuota.sizeProcessedMessages = pickMaxUint64(maxQuota.sizeProcessedMessages, q.sizeProcessedMessages)
+		log.Trace("peer quota statistics",
+			"peer", name,
+			"num received msg", q.numReceivedMessages,
+			"size received", core.ConvertBytes(q.sizeReceivedMessages),
+			"num processed msg", q.numProcessedMessages,
+			"size processed", core.ConvertBytes(q.sizeProcessedMessages),
+		)
+	}
+
+	log.Trace("quota statistics", "num peers", numStatistics)
+	log.Trace("min quota statistics / peer",
+		"num received msg", minQuota.numReceivedMessages,
+		"size received", core.ConvertBytes(minQuota.sizeReceivedMessages),
+		"num processed msg", minQuota.numProcessedMessages,
+		"size processed", core.ConvertBytes(minQuota.sizeProcessedMessages),
+	)
+	log.Trace("avg quota statistics / peer",
+		"num received msg", avgQuota.numReceivedMessages/uint32(numStatistics),
+		"size received", core.ConvertBytes(avgQuota.sizeReceivedMessages/uint64(numStatistics)),
+		"num processed msg", avgQuota.numProcessedMessages/uint32(numStatistics),
+		"size processed", core.ConvertBytes(avgQuota.sizeProcessedMessages/uint64(numStatistics)),
+	)
+	log.Trace("max quota statistics / peer",
+		"num received msg", maxQuota.numReceivedMessages,
+		"size received", core.ConvertBytes(maxQuota.sizeReceivedMessages),
+		"num processed msg", maxQuota.numProcessedMessages,
+		"size processed", core.ConvertBytes(maxQuota.sizeProcessedMessages),
+	)
+	log.Trace("total quota statistics / network",
+		"num received msg", avgQuota.numReceivedMessages,
+		"size received", core.ConvertBytes(avgQuota.sizeReceivedMessages),
+		"num processed msg", avgQuota.numProcessedMessages,
+		"size processed", core.ConvertBytes(avgQuota.sizeProcessedMessages),
+	)
+}
+
+func pickMinUint32(val1 uint32, val2 uint32) uint32 {
```

Comment:
instead of these helper functions you can use the existing ones from `core/computers.go`

Reply:
done